package.json

```diff
@@ -36,10 +36,10 @@
     "mobx": "^3.6.2",
     "mobx-react": "^4.4.3",
     "qr-image": "^3.2.0",
-    "react": "~16.3.2",
-    "react-art": "~16.3.2",
-    "react-dom": "~16.3.2",
-    "react-native-web": "^0.6.0",
+    "react": "^16.5.2",
```

Comment:
Don't think these changes should be part of this PR?

Reply:
Because `stickyHeaderIndices` for `ListView` were buggy I had to upgrade `react-native-web` and its dependencies `react`, `react-art`, `react-dom`. Should we make the upgrade in a separate PR and rebase this one once the upgrade PR is merged?

Reply:
> Don't think these changes should be part of this PR?

This is tricky. I've been putting off upgrading react/react-native-web since there was quite a bit of breakage in some views. @erkarl have you tested all screens in the storybook to make sure there are no regressions? I remember ListViews not scrolling without a manual window resize after items were added dynamically by actions.

Reply:
Good point - I'll make sure to test this when I make the other requested changes in this PR.

Reply:
@tanx I tested the ListView scrolling by dynamically adding items - didn't see any issues.

As for testing the screens for visual regressions - I did test that when putting up the PR. The only regression I found was with the `seed` view which is fixed in this PR.

Please double test 🙏 .

Reply:
I tested the new dependency version. Looks like it breaks the seed view in storybook.

Reply:
This the output I'm getting in `erkarl:sticky-list-headers` branch:
<img width="1420" alt="asdf" src="https://user-images.githubusercontent.com/1056569/46477170-b0619b80-c7f2-11e8-956f-55bf1987ed2c.png">

Can you elaborate on what's broken here?

Reply:
This is on my machine after installing the versions in the package.json

<img width="1667" alt="bildschirmfoto 2018-10-04 um 15 37 43" src="https://user-images.githubusercontent.com/1374174/46477689-80fb6080-c7eb-11e8-8d73-3d07efd63382.png">

Reply:
Ok I upgrade react/react-native-web in a separate PR and merged to master. @erkarl would you kindly rebase this PR on top of the master? thanks